src/types/basic/abstract.ts

```diff
@@ -20,36 +19,15 @@ export function isTypeOf(type: unknown, typeSymbol: symbol): boolean {
  *
  * It is serialized as, at maximum, 32 bytes and merkleized as, at maximum, a single chunk
  */
-export class BasicType<T> {
-  /**
-   * Symbols used to track the identity of a type
-   *
-   * Used by various isFooType functions
-   */
-  _typeSymbols: Set<symbol>;
-
+export abstract class BasicType<T> extends Type<T> {
   constructor() {
```

Comment:
Can remove constructor

Reply:
Yup. I've put it because of wierd constructor error. But it was unrelated